__tests__/pages/notifications/deleteSearch.test.js

```diff
@@ -21,15 +21,17 @@ jest.mock('../../../src/service/api-key-service');
 jest.mock('../../../src/utils/contentFulPage');
 jest.mock('../../../src/utils/cookieAndJwtChecker');
 jest.mock('../../../src/service/saved_search_service');
+jest.mock('next/config', () => () => ({
+  publicRuntimeConfig: {},
+}));
```

Comment:
If this is repeated a lot, can setup jest to run it before each test using some config